Framework/Crystal/src/SaveHKL.cpp

```diff
@@ -233,22 +233,22 @@ void SaveHKL::exec() {
   }
 
   if (correctPeaks) {
-    std::vector<double> spec(11);
-    std::string STRING;
     std::ifstream infile;
     std::string spectraFile = getPropertyValue("SpectraFile");
     infile.open(spectraFile.c_str());
     if (infile.is_open()) {
+      std::string ignoreLine;
       size_t a = 1;
       for (int wi = 0; wi < 8; wi++)
-        getline(infile, STRING); // Saves the line in STRING.
-      while (!infile.eof())      // To get you all the lines.
+        getline(infile, ignoreLine); // Saves the line in STRING.
```

Comment:
```suggestion
        getline(infile, ignoreLine); // Saves the line in ignoreLine.
```

Reply:
Thanks. I've just removed this comment altogether since these lines are really "saved" anyway.